cdap-ui/app/features/tracker/controllers/entity-ctrl.js

```diff
@@ -25,27 +25,65 @@ class TrackerEntityController{
     this.datasetType = rDatasetType;
     this.datasetExplorable = Array.isArray(rSystemTags) && rSystemTags.indexOf('explore') !== -1;
 
+    // Mock data
+    this.truthMeterData = {
+      datasets: [
+        {
+          name: 'history',
+          value: 80
+        },
+        {
+          name: 'purchases',
+          value: 26
+        }
+      ],
+      streams: [
+        {
+          name: 'purchaseStream',
+          value: 44
+        },
+        {
+          name: 'strm2',
+          value: 99
+        }
+      ]
+    };
+
     switch (entitySplit[0]) {
       case 'streams':
         this.entityInfo = {
           name: 'Stream',
           icon: 'icon-streams'
         };
+        for (var i=0; i < this.truthMeterData.streams.length; i++) {
+          if(this.truthMeterData.streams[i].name === this.$state.params.entityId) {
+            this.entityInfo.score = this.truthMeterData.streams[i].value;
+          }
+        }
         break;
       case 'datasets':
         this.entityInfo = {
           name: 'Dataset',
           icon: 'icon-datasets'
         };
+        for (var n=0; n < this.truthMeterData.datasets.length; n++) {
+          if(this.truthMeterData.datasets[n].name === this.$state.params.entityId) {
+            this.entityInfo.score = this.truthMeterData.datasets[n].value;
+          }
+        }
         break;
       case 'views':
         this.entityInfo = {
           name: 'Stream View',
           icon: 'icon-streams'
         };
+        for (var x=0; x < this.truthMeterData.streams.length; x++) {
+          if(this.truthMeterData.streams[x].name === this.$state.params.entityId) {
+            this.entityInfo.score = this.truthMeterData.streams[x].value;
+          }
+        }
```

Comment:
after speaking with @elfenheart this is going to stay for now, and it can be removed once we're 100% sure